tests/networks.spec.ts

```diff
@@ -14,16 +14,20 @@ import {
   setOption,
 } from "./helpers/configuration";
 
-test("network create and remove", async ({ page }) => {
-  const network = randomNetworkName();
+let network = randomNetworkName();
+let page: Page;
+test.beforeAll(async ({ browser, browserName }) => {
+  network = `${browserName.substring(0, 2)}-${network}`;
```

Comment:
Why are we using only a substring of the browser name here, but the full string for instances, profiles and storage volumes? Is that because of some stricter limit on the length of the network name?

Reply:
yup, network names can only be 15 characters long. I chose to substring the browser name instead of the network name since `randomNetworkName` uses `randomNameSuffix` that's also used by many other tests e.g. storage, instance etc. I think best I leave a comment here for context, I see why this is confusing